cmd/utils.go

```diff
@@ -72,6 +72,33 @@ func checkDuplicates(list []string) error {
 	return nil
 }
 
+// checkDuplicates - function to validate if there are duplicates in a slice of strings.
+func checkDuplicateEndPoints(list []storageEndPoint) error {
+	// Empty lists are not allowed.
+	if len(list) == 0 {
+		return errInvalidArgument
+	}
+	// Empty end points are not allowed.
+	for _, ep := range list {
```

Comment:
why do this? can checkDuplicateStrings be used here as well? since you already have "ep.String()" method.